avalanche/training/strategies/deep_slda.py

```diff
@@ -0,0 +1,287 @@
+import warnings
+from typing import Optional, Sequence
+
+import os
+import torch
+from torch import nn
+import torchvision.models as models
+
+from avalanche.training.plugins import StrategyPlugin
+from avalanche.training.strategies import BaseStrategy
+from avalanche.training import default_logger
+from avalanche.models.dynamic_modules import MultiTaskModule
+from avalanche.models import ModelWrapper
+
+
+class SLDAResNetModel(nn.Module):
```

Comment:
I'd move this class to the models package...

Reply:
If we believe this to be a model useful also in other cases I can move it there. It currently uses `pool_feat` which is something a bit too specific in my opinion. Its purpose is to reproduce SLDA results, not to be useful at a general level. 
It really depends on the purpose of `models` module, I think.

Reply:
In "models" we have already specific models used by other strategies... maybe we can have a specific sub-package there for this kind of models. But I'd prefer to maintain all the models in a single place.